lifecycle-operator/apis/lifecycle/v1alpha3/keptnappversion_conversion_test.go

```diff
@@ -51,18 +51,6 @@ func TestKeptnAppVersion_ConvertFrom(t *testing.T) {
 								Version: "4.5.6",
 							},
 						},
-						PreDeploymentTasks: []string{
```

Comment:
Please revert changes in this file, the tests should not be removed, but adapted